package.json

```diff
@@ -131,6 +131,7 @@
     }
   },
   "devDependencies": {
+    "@arethetypeswrong/cli": "^0.15.3",
```

Comment:
This dependency does add some weight to installed packages. I don't remember the numbers printed by Yarn, but with npm it takes:

- 37 packages
- 35 MB on disk

Reply:
Great comment! It might seem a little big, but I guess it's worth it, and it's devDependencies anyway.